app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -261,12 +267,21 @@ const ListContainer = (props: {
   }, [params?.options?.events, params?.search, params?.type])
 
   return (
-    <Theme>
-      <SafeArea>
-        <TopOfApp
-          showSearchButton
-          onSearchButtonPress={() => navigate(screens.search[params.type], '')}
-        />
+    <>
+      <TopOfApp
+        showSearchButton
+        onSearchButtonPress={() => navigate(screens.search[params.type], '')}
+      />
+      <VStack
+        _dark={{ backgroundColor: 'bgDarkMode.100' }}
+        _light={{ backgroundColor: 'bg.secondary' }}
+        paddingBottom="2"
+        alignItems="center"
+        space={4}
+        padding={4}
+      >
+        <Heading fontSize="md">Projects List</Heading>
```

Comment:
From looking at [the NativeBase docs for Heading](https://docs.nativebase.io/heading), I think the recommended way to change a heading is using the `size` attribute rather than `fontSize` (works differently to e.g. `<Text>`